### DISCUSSION TITLE
编译原理总结 | Go 语言设计与实现 · /ch02-summary

### DISCUSSION THREAD

**Discussion by draveness** [draveness/blog-comments#582] [Announcements]
https://draveness.me/ch02-summary

**Comment by lupguo** [comment]
很赞！

**Comment by zhangxu-ai** [comment]
写的太好了！

**Comment by yanjinbin** [comment]
楼主 能不能说说 用什么工具,去查看一个示例代码编译之后,从程序最原始入口起点开始分析编译后的代码对应源码过程的教程哦?

**Comment by yanjinbin** [comment]
我虽然跟着你的分析 你看懂一部分 但是 我更想自己学着像你这样分析源码如何被编译,比如for range 编译之后底层用的什么方法实现关键字的语义

**Comment by draveness** [comment]
> 我虽然跟着你的分析 你看懂一部分 但是 我更想自己学着像你这样分析源码如何被编译,比如for range 编译之后底层用的什么方法实现关键字的语义

第一章会介绍这个，等等吧
